tests/system/providers/arangodb/example_arangodb.py

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from datetime import datetime
+
+from airflow.models.dag import DAG
+from airflow.providers.arangodb.operators.arangodb import AQLOperator
+from airflow.providers.arangodb.sensors.arangodb import AQLSensor
+from tests.system.utils import get_test_env_id
+
+ENV_ID = get_test_env_id()
+DAG_ID = "example_arangodb_operator"
+
+with DAG(
+    DAG_ID,
+    start_date=datetime(2021, 1, 1),
+    tags=['example'],
```

Comment:
```suggestion
    schedule_interval='@once',
    tags=['example'],
```
Looks like [this parameter is explicitly needed](https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-47+New+design+of+Airflow+System+Tests#:~:text=Make%20sure%20to%20include%20these%20parameters%20into%20DAG%20call%3A) based on the AIP docs.